### DISCUSSION TITLE
Survey

### DISCUSSION THREAD

**Discussion by Natashka333** [timmyichen/dev-directory#42] [Ideas]
hey guys, @wo1vin  and I worked on a survey for the app. The goals of survey are: to get know users' needs and to understand what we should prioritize while building. The survey is open for commenting (please, feel free to leave your feedbacks, or concerns). Thanks!

[https://docs.google.com/document/d/14TScmbucvAygA7j8I3Hq9uIX2fvusUXf_-hV_JlLngI/edit?usp=sharing](https://docs.google.com/document/d/14TScmbucvAygA7j8I3Hq9uIX2fvusUXf_-hV_JlLngI/edit?usp=sharing)

**Comment by wo1vin** [comment]
We got a decent amount of great feedback to work with. I like your introduction and insightful questions. Did you want to interview people using the google doc, or would you like us to make the google form into the final survey?

**Comment by Natashka333** [comment]
Thank you for the feedback and your help. I like your Google form which you created couple days ago. I just need to edit the document and we can push questions there. 🤓

**Comment by DevinCLane** [comment]
Added some edits, looking good so far!

**Reply by Natashka333** [reply]
I appreciate it! Thanks ;)

**Comment by Natashka333** [comment]
Survey results :tada:
[100Devs Directory Survey.xlsx](https://github.com/timmyichen/dev-directory/files/11067038/100Devs.Directory.Survey.xlsx)
